pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -394,9 +392,7 @@ void createNameSpace(String cluster, String publicTenant, NamespaceName ns) thro
         }
 
         if (!nsr.namespaceExists(ns)) {
-            Policies nsp = new Policies();
-            nsp.replication_clusters = Collections.singleton(config.getClusterName());
-            nsr.createPolicies(ns, nsp);
+            broker.getAdminClient().namespaces().createNamespace(ns.toString());
```

Comment:
Sorry, I don't get it, why `nsr.createPolicies(ns, nsp);` is ignored?

Reply:
Some policy is ignored because the conf file effects this policy.